README.md

```diff
@@ -157,6 +157,12 @@ To skip cleaning the project:
 snapshot --noclean
 ```
 
+To only run tests (i.e. don't actually take any screenshots):
```

Comment:
Yep. Wasn't sure how I could disable the actual screenshot taking? Would redefining the function that does it to be a no-op work?